package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gpii-pouch",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "node src/tests/all-tests.js"
+  },
+  "dependencies": {
+    "express":         "4.10.5",
+    "express-pouchdb": "0.14.0",
+    "infusion":        "git://github.com/fluid-project/infusion.git#df5f8cfabf815a4086b778e73125c3c952dda4ec",
+    "memdown":         "1.0.0",
+    "pouchdb":         "3.0.5",
```

Comment:
On further reflection I really assume you mean that I should update to make the windows build more stable.  I confirmed that the tests work with 3.3.1 and committed the change to `package.json`.